src/afl-fuzz-extras.c

```diff
@@ -305,10 +305,14 @@ static inline u8 memcmp_nocase(u8 *m1, u8 *m2, u32 len) {
 }
 
 /* Maybe add automatic extra. */
+/* Ugly hack: afl state is transfered as u8* because we import data via
```

Comment:
I suggest making this a `void *data` ptr, not `u8 *`. I feel that, semantically, u8 is wrong here. It's not super ugly, we actually may be able to reuse this for the coming custom lib refactorings.
For the future, do we want to add a meta afl struct for all shared env vars between tools? Right now, they are still global in afl-showmap et al. but that doesn't make sense imho.